src/OSPSuite.Core/Domain/Formulas/DiscreteDistributionFormula.cs

```diff
@@ -13,7 +13,8 @@ public DiscreteDistributionFormula() : base(DistributionType.Discrete)
       protected override double CalculateFor(IReadOnlyList<IObjectReference> usedObjects, IUsingFormula dependentObject)
       {
          var distributedParameter = dependentObject.ConvertedTo<IDistributedParameter>();
-         return CalculateValueFromPercentile(distributedParameter.Percentile, distributedParameter);
+         //percentile not used in this formula
+         return CalculateValueFromPercentile(Constants.DEFAULT_PERCENTILE, distributedParameter);
```

Comment:
the parameter is not used. might was well not retrieve it (not related to this error)